niworkflows/anat/mni.py

```diff
@@ -17,28 +17,43 @@
 from niworkflows import __packagename__, NIWORKFLOWS_LOG
 
 class RobustMNINormalizationInputSpec(BaseInterfaceInputSpec):
+    # Set the input file to be used as the moving image.
     moving_image = InputMultiPath(
         File(exists=True), mandatory=True, desc='image to apply transformation to')
+    # Set an input file to be used as the reference image (instead of the default template).
     reference_image = InputMultiPath(
         File(exists=True), desc='override the reference image')
+    # Set the input file to be used as the moving mask.
     moving_mask = File(exists=True, desc='moving image mask')
+    # Set the input file to be used as the reference mask.
     reference_mask = File(exists=True, desc='reference image mask')
+    # Set the input file to be used as the lesion mask.
+    lesion_mask = File(exists=True, desc='lesion mask image')
```

Comment:
Could you specify which image the lesion mask corresponds to (moving or reference)?